### PR TITLE
Solana pay [Reference]

### PR DESCRIPTION
**Pull request by 0xPratik**

### CODE REVIEW

**Comment by 0xPratik** [comment]
Hey @jacobcreech do you think there should be an instruction on how to setup Point of Sale Application in the cookbook

**Comment by jacobcreech** [comment]
The PoS app tutorial could be completely separate and should probably lie outside of the cookbook as a standalone tutorial

File: docs/.vuepress/config.ts [hunk 1]
```diff
@@ -1,127 +1,128 @@
-import { defineUserConfig } from 'vuepress';
```
Comment (jacobcreech) [inline docs/.vuepress/config.ts:1]: Could you remove the reformatting?

File: docs/references/solana-pay.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+---
+
+# Solana Pay
+
+**@solana/pay** is a JavaScript library for facilitating commerce on Solana by using a token transfer URL scheme. The URL scheme ensures that no matter the wallet or service used, the payment request must be created and interpreted in one standard way.
```
Comment (jacobcreech) [inline docs/references/solana-pay.md:41]: Could we link the [scheme](https://docs.solanapay.com/spec) here?

File: docs/references/solana-pay.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## How to receive payments with Solana pay
+
+To start getting payments using Solana pay you need to have a native SOL address. This address is going to be the merchant address which is going to get funded whenever a user pays using solana pay.
+
+Let's Look into how you can integrate Solana Pay in your Application and start accepting SOL.
+
+### Creating a SOL Payment request
+
+In this Code-snippet we are going to look into how create a payment request URL and encode it into a QR Code
```
Comment (jacobcreech) [inline docs/references/solana-pay.md:51]:
```suggestion
To create a SOL payment request, we first generate a QR code with a payment request URI.
```

File: docs/references/solana-pay.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+  </template>
+
+  </SolanaCodeGroupItem>
+
+</SolanaCodeGroup>
+
+### Creating a SPL Token payment request
+
+In this Code-snippet we are going to look into how create a payment request for a user who can transfer tokens of a specific mint accepted by the merchant.
```
Comment (jacobcreech) [inline docs/references/solana-pay.md:74]:
```suggestion
To create a payment request using a custom token other than Native SOL, you can specify the mint address in the payment request URL.
```

File: docs/references/solana-pay.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+      content: Solana Cookbook | Solana Pay Reference
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | Solana Pay Reference
+  - - meta
+    - name: description
+      content: Learn how to use Solana Pay
+  - - meta
+    - name: og:description
+      content: Learn how to write programs on Solana, with references on cross program invocation, reading accounts, and more
```
Comment (jacobcreech) [inline docs/references/solana-pay.md:15]: I don't think this `og:description` matches what is on the page.

File: docs/references/solana-pay.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+---
+title: Solana Pay
+head:
+  - - meta
+    - name: title
+      content: Solana Cookbook | Solana Pay Reference
```
Comment (jacobcreech) [inline docs/references/solana-pay.md:6]:
```suggestion
      content: Solana Cookbook | Solana Pay
```

File: docs/references/solana-pay.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | Solana Pay Reference
```
Comment (jacobcreech) [inline docs/references/solana-pay.md:9]:
```suggestion
      content: Solana Cookbook | Solana Pay
```

**Review by jacobcreech — CHANGES_REQUESTED** [review]
A few changes. We may want to also move this over to `Integrations` instead of the main page of cookbook references.

**Comment by 0xPratik** [comment]
Hey @jacobcreech I have done the requested changes according to the review. Should we move this section to integrations currently it's in references

**Comment by 0xPratik** [comment]
Hello @jacobcreech I have moved the Solana Pay section to integrations

**Comment by jacobcreech** [comment]
@0xPratik If you're leaving in all of the format changes on the config, could you also add whatever prettier or lint config you have? Would fall under #280

**Comment by 0xPratik** [comment]
Sure Ser Will add that as well and I was thinking of adding a section of transaction request's as well in the Solana Pay integration section once it gets merged in the master branch of Solana pay or Should i add it later in a different PR

**Comment by jacobcreech** [comment]
@0xPratik a different PR for transaction request would be fine

**Comment by 0xPratik** [comment]
Hey @jacobcreech can you please review this PR

**Pull request closed without merging**